docs/build/html/about.html

```diff
@@ -311,8 +311,8 @@ <h3>3. Map skills<a class="headerlink" href="#map-skills" title="Permalink to th
 </div>
 <div class="section" id="app">
 <h2>App<a class="headerlink" href="#app" title="Permalink to this heading">#</a></h2>
-<p>If you would like to demo the library using a front end, we have also <a class="reference external" href="http://18.169.52.145:8501/">built a streamlit app</a> that allows you to extract skills for a given text. The app allows you to paste a job advert of your choice, extract and map skills onto any of the configurations: <code class="docutils literal notranslate"><span class="pre">extract_skills_toy</span></code>, <code class="docutils literal notranslate"><span class="pre">extract_skills_lightcast</span></code> and <code class="docutils literal notranslate"><span class="pre">extract_skills_esco</span></code>.</p>
-<p><img alt="nesta_esco" src="https://user-images.githubusercontent.com/46863334/206517833-fdf3c7f4-b90e-4dc9-94ca-4d6710abae6f.gif" /></p>
+<p>If you would like to demo the library using a front end, we have also <a class="reference external" href="http://13.42.37.54:8501/">built a streamlit app</a> that allows you to extract skills for a given text. The app allows you to paste a job advert of your choice, extract and map skills onto any of the configurations: <code class="docutils literal notranslate"><span class="pre">extract_skills_lightcast</span></code> and <code class="docutils literal notranslate"><span class="pre">extract_skills_esco</span></code>.</p>
```

Comment:
change to https://nesta-skills-extractor.dap-tools.uk/

Reply:
change to `https://www.nesta.org.uk/data-visualisation-and-interactive/skills-extractor-tool/`